spec/lucky/param_serializable_spec.cr

```diff
@@ -0,0 +1,199 @@
+require "../spec_helper"
+
+include ContextHelper
+include MultipartHelper
+
+class BasicParams
+  include Lucky::ParamSerializable
+  skip_param_key
+
+  property string : String
+  property int16 : Int16
+  property int32 : Int32
+  property int64 : Int64
+  property bool : Bool
+  property float64 : Float64
+  property uuid : UUID
+  property blank : String?
+end
+
+class UserWithKeyParams
+  include Lucky::ParamSerializable
+  param_key :user
```

Comment:
Personally, I prefer this being explicit rather than implicit. Then `skip_param_key` would not be necessary because that would be the default